src/server/scripts/Northrend/zone_grizzly_hills.cpp

```diff
@@ -612,6 +617,14 @@ class npc_venture_co_straggler : public CreatureScript
             if (!UpdateVictim())
                 return;
 
+	    if (Chop_Timer <= diff)
+	    {
+		DoCastVictim(SPELL_CHOP);
+		Chop_Timer = 8000;
+	    }
+	    else
+		Chop_Timer -= diff;
```

Comment:
A lot of tab, dont use tab, change it for 4 spaces.

Reply:
will remember it thanks